src/components/ContextMenuItem.js

```diff
@@ -47,62 +46,53 @@ const defaultProps = {
     description: '',
 };
 
-class ContextMenuItem extends Component {
-    constructor(props) {
-        super(props);
+function ContextMenuItem(props) {
```

Comment:
@gedu You can use destructuring now -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring

How about destructuring all props here?

Reply:
@fabioh8010 They updated this rule? As I remember it was restricted.

Reply:
It's updated in the docs already, so I guess the eslint rule is updated as well.